src/foundry/client/data/documents/cards.d.mts

```diff
@@ -337,13 +346,6 @@ declare global {
      */
     resetDialog(): Promise<Cards.ConfiguredInstance | false | null>;
 
-    override deleteDialog(options?: Partial<DialogOptions>): Promise<this | false | null | undefined>;
-
-    // TODO: It's a bit weird that we have to do it in this generic way but otherwise there is an error overriding this. Investigate later.
-    static override createDialog<T extends DocumentConstructor>(
-      this: T,
-      data?: DeepPartial<Cards["_source"] | (Cards["_source"] & Record<string, unknown>)>,
-      context?: Pick<DocumentModificationContext, "parent" | "pack"> & Partial<DialogOptions>,
-    ): Promise<InstanceType<ConfiguredDocumentClass<T>> | null | undefined>;
+    // deleteDialog and createDialog have the same signatures as the parent class
```

Comment:
I mentioned this on another PR but technically the stance is still to copy from the parent when there's no signature change.

I'm thinking about the impacts of this opinion on performance occasionally though but the idea was that it'd make diffing easier.

For now put it back please, we'll want something more standard for diffing purposes anyways